File: backend/travel-with-anna/src/main/java/com/andwis/travel_with_anna/trip/day/activity/ActivityDetailedResponse.java
package com.andwis.travel_with_anna.trip.day.activity;

import com.andwis.travel_with_anna.address.AddressDetail;

import java.math.BigDecimal;
import java.util.List;

public record ActivityDetailedResponse(
        AddressDetail addressDetail,
        List<ActivityResponse> activities,
        BigDecimal totalPrice,
        BigDecimal totalPayment
) {
}

File: frontend/travel-with-anna-ui/src/app/services/models/activity-detailed-response.ts
/* tslint:disable */
/* eslint-disable */
import { ActivityResponse } from '../models/activity-response';
import { AddressDetail } from '../models/address-detail';
export interface ActivityDetailedResponse {
  activities?: Array<ActivityResponse>;
  addressDetail?: AddressDetail;
  totalPayment?: number;
  totalPrice?: number;
}
